lib/hawkular/metrics/metric_api.rb

```diff
@@ -40,6 +40,28 @@ def push_data(gauges: [], counters: [], availabilities: [])
       http_post(path, data)
     end
 
+    # Fetch stats for multiple metrics of all supported types
+    # @param gauge_ids [Array[String]] list of gauge ids
+    # @param counter_ids [Array[String]] list of counter ids
+    # @param avail_ids [Array[String]] list of availability ids
+    # @param starts [Integer] optional timestamp (default now - 8h)
+    # @param ends [Integer] optional timestamp (default now)
+    # @param bucket_duration [String] optional interval (default 3600s)
+    # @return [Hash] stats grouped per type
+    # @example
+    #   client = Hawkular::Metrics::client::new
+    #   metrics = { gauge: ['G1', 'G2'], counter: ['C2', 'C3'], availability: ['A2', 'A3'] }
+    #   starts = 200
+    #   ends = 500
+    #   bucketDuration = '150ms'
+    #   client.query_stats()
```

Comment:
The example seems to be a bit confusing, query_stats expect gauge_ids, counter_ids, avail_ids but you start to group them on a metrics hash. Could you make it a bit clearer?